tailwind.config.ts

```diff
@@ -17,4 +26,6 @@ export default {
       },
     },
   },
+  darkMode: ['variant', '&:not(.light *)'],
```

Comment:
```suggestion
  darkMode: ['variant'],
```
this need to be simplified, ping me if you need help

Reply:
Does the write-up I provided in another pull request close this topic?

This is the simplest implementation for the feature we need. It's even in the docs.

https://tailwindcss.com/docs/dark-mode#overriding-the-dark-variant

Reply:
lgtm